CMakeLists.txt

```diff
@@ -312,4 +315,10 @@ add_subdirectory(test)
 
 if(TRITON_BUILD_UT)
   add_subdirectory(unittest)
+  # This target runs all the unit tests.
+  add_custom_target(check-triton-unit
```

Comment:
Some bikeshedding, how about `run-TritonUnitTests`?

Reply:
I chose this name because it lines up with `check-triton-lit-tests`, although by that logic, I should have named it `check-triton-unit-tests` :P

Reply:
Ah didn't know that existed haha. `unit-tests` does sound better though

Reply:
@peterbell10 I changed it to `check-triton-unit-tests`. PTAL :)